PC02_Graffiti_EPPERSON.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Max Epperson
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+home()
+showturtle()
+up()
+clear()
+goto(120,-90)
+clear()
+
```

Comment:
Add dot notation to the block of code that creates the piss jar base 
```suggestion
#define common variables for this section
Rrotation = 45 #right rotation
Lrotation = 90 #left rotation
Shortside = 50
Longside = 100

#draws the yellow base of the piss jar 
jarTurt.width(5)
jarTurt.down()
jarTurt.fillcolor(pissyellow)
jarTurt.begin_fill()
jarTurt.left(Lrotation)
jarTurt.forward(Shortside)
jarTurt.right(Rrotation)
jarTurt.goto(130,-30)
jarTurt.right(Rrotation)
jarTurt.forward(Longside)
jarTurt.right(Rrotation)
jarTurt.goto(240,-40)
jarTurt.right(Rrotation)
jarTurt.forward(Longside+50)
jarTurt.right(Rrotation)
jarTurt.goto(230, -200)
jarTurt.right(Rrotation)
jarTurt.forward(Longside)
jarTurt.right(Rrotation)
jarTurt.goto(120,-190)
jarTurt.right(Rrotation)
jarTurt.forward(Longside)
jarTurt.end_fill()
```